### DISCUSSION TITLE
Inaccurate Org Membership

### DISCUSSION THREAD

**Discussion by eljeffeg** [lowlighter/metrics#690] [Q&A] [answered]
I'm a member of an organization ([SGI](https://github.com/StargateInvasion)), but the metric shows 0.

**Comment by spenserblack** [comment, marked as answer]
[This](https://github.com/orgs/StargateInvasion/people) currently says that you're not a public member. Is your membership private?
You may want to reference [these docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership).
